hack/run-bdd-suite.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/sh
+# Example: ./hack/run-bdd-suite.sh tests/bdd-smoke/suites/config/libvirt
+
+if [ "$IS_CONTAINER" != "" ]; then
+  mkdir ~/.ssh
+  echo "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAAgQCc4amSo5Z59mH4bUgZ4M8A1sURg+qUKkYjZ9+0fft5/OmSbe34Ge9gUrJ8UKNpmbC+W26v9qKX21NCYct1QOgIlVqyIjZUll0NfcJXToAd1o2GrItMlpQEpXfQGNOjZtWu1uyO187qbGMwnq8e8EH9R1IJZwTI6sOVnc8sSyzydQ==" > ~/.ssh/id_rsa.pub
```

Comment:
hmm, what is this for ?

Reply:
this is to add a valid ssh test key inside of the golang's container and then validate it by the framework.